compiler/one-cmds/one-codegen

```diff
@@ -94,15 +95,40 @@ def _get_parser(backends_list):
     return parser
 
 
-def _verify_arg(parser, args):
+def _verify_arg(parser, args, cfg_args):
     """verify given arguments"""
+    cmd_backend_exist = oneutils.is_valid_attr(args, 'backend')
+    cfg_backend_exist = oneutils.is_valid_attr(cfg_args, 'backend')
+    cfg_backends_exist = oneutils.is_valid_attr(cfg_args, 'backends')
+    if cfg_backend_exist and cfg_backends_exist and oneutils.is_valid_attr(
```

Comment:
(Q) Can't we use `backend` and `backends` options at the same time in cfg?

Reply:
Yeah, I intentionally made it impossible to make the implementation simple. Otherwise, it'd too complicated. I think there is no need to use them at the same time.

Reply:
(Q) Is there no case to share command key between multiple backends?

Reply:
Which case do you think it better be supported? As I said offline, I think we don't need to support such cases.